tests/datavolume_test.go

```diff
@@ -3393,15 +3393,30 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			Expect(err).ToNot(HaveOccurred())
 
 			By("verifying PVC was created")
-			_, err = utils.WaitForPVC(f.K8sClient, sourceDataVolume.Namespace, sourceDataVolume.Name)
+			pvc, err := utils.WaitForPVC(f.K8sClient, sourceDataVolume.Namespace, sourceDataVolume.Name)
+			Expect(err).ToNot(HaveOccurred())
+			f.ForceBindIfWaitForFirstConsumer(pvc)
+
+			By("Wait for DV to succeed")
+			err = utils.WaitForDataVolumePhaseWithTimeout(f, sourceDataVolume.Namespace, cdiv1.Succeeded, sourceDataVolume.Name, 10*time.Minute)
```

Comment:
I believe (correct me if this is wrong) that these tests were purposely written like this since the labels would show prior to the DV succeeding

Reply:
For registry imports waiting is required. All others could work without. Would you like me to change it so only the registry import waits?

Reply:
Yeah alongside a comment, that would be great